### PR TITLE
Fix miss spelling on TSK

### PR DESCRIPTION
**Pull request by master-q**

### CODE REVIEW

File: documentation/secure_storage.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ per-device keys for different subsystems using the same algorithm as we
 generate the SSK; An easy way to generate different per-device keys for
 different subsystems is using different static strings to generate the keys.
 
-### Trusted Application Storage Key (TKS)
+### Trusted application Storage Key (TSK)
```
Comment (jbech-linaro) [inline documentation/secure_storage.md]: Application: Grammatically you're right, but it seems like we always write TA as **T**rusted **A**pplication.

TSK: Thanks!

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by jforissier** [comment]
@master-q can you please:
- write "Trusted Application" as requested by @jbech-linaro,
- rebase on top of master,
- add your `Signed-off-by:` tag to the commit description, as well as @jbech-linaro's `Reviewed-by:`.

Then I can merge this.

**Comment by jbech-optee-adm** [comment]
Closing the PR since it has fixed and been merged manually.

**Pull request closed without merging**